### PR TITLE
Added new carousel component

### PR DESCRIPTION
**Pull request by pablo3443**

### CODE REVIEW

File: src/layout/carousel/CarouselDotsView.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import DotView from './DotView';
+
+export default Marionette.CollectionView.extend({
+    tagName: 'div',
```
Comment (stasguryev) [inline src/layout/carousel/CarouselDotsView.ts]: -

File: src/layout/carousel/CarouselDotsView.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    className: 'carousel__dots',
+
+    childView: DotView,
+
+    updateActive(currentIndex) {
+        const isActive = index => currentIndex <= index && index < currentIndex + this.options.slidesToShow;
+        this.children.forEach(child => child.$el.toggleClass('carousel__dot_active', isActive(child.model.id)));
```
Comment (stasguryev) [inline src/layout/carousel/CarouselDotsView.ts]: chilren.forEach(child => updateActive())

File: src/layout/carousel/CarouselDotsView.ts (hunk 1 above)
Comment (pablo3443) [reply src/layout/carousel/CarouselDotsView.ts]: done

File: src/layout/carousel/CarouselDotsView.ts (hunk 2 above)
Comment (pablo3443) [reply src/layout/carousel/CarouselDotsView.ts]: done

**Pull request merged**